pkg/services/action.go

```diff
@@ -233,6 +234,15 @@ func NewSubmitBusinessCase(
 			appcontext.ZLogger(ctx).Error("Submit Business Case email failed to send: ", zap.Error(err))
 		}
 
+		// TODO - EASI-2363 - rework conditional to also trigger on publishing finalized system intakes
+		// need to check intake.Status, *not* businessCase.SystemIntakeStatus - intake is what gets returned from calling updateIntake()
```

Comment:
Is this also because we haven't updated the status of the intake yet, which seems to happen a few lines up?

```golang
		intake.Status = newIntakeStatus
```

Reply:
Yes, that's another reason; if `intake.Status` was updated and `updateIntake()` was called before `updateBusinessCase()`, then `businessCase.SystemIntakeStatus` would have the new status. As it is, `updateBusinessCase()` is called before the intake status is changed, so `businessCase.SystemIntakeStatus` will have the old status.

Reply:
No, we don't send it to the CEDAR Intake API. I didn't see a need to, given that we send the intake ID on the business case; I figure CEDAR/Alfabet can use that to look up the system intake's status.